tools/clang/unittests/HLSLHost/HLSLHost.cpp

```diff
@@ -827,16 +821,16 @@ class ServerFactory : public IClassFactory {
 public:
   DXC_MICROCOM_ADDREF_RELEASE_IMPL(m_dwRef)
   ServerFactory() : m_dwRef(0) {}
-  HRESULT STDMETHODCALLTYPE QueryInterface(REFIID iid, void **ppvObject) {
+  HRESULT STDMETHODCALLTYPE QueryInterface(REFIID iid, void **ppvObject) override {
```

Comment:
For these override keywords, what warning went away when adding override?

Reply:
https://clang.llvm.org/docs/DiagnosticsReference.html#winconsistent-missing-override